### DISCUSSION TITLE
Button opacity options

### DISCUSSION THREAD

**Discussion by LostHati** [dnwjn/youtube-watch-later#34] [Ideas]
### Idea

A setting to change the opacity of the button on thumbnails so it doesn't block a part of it.  Another cool feature would be a setting that only shows the button on thumbnail-hover.
I tried to do it myself but can't seem to change the button's color strangely.

**Comment by dnwjn** [comment]
Hi @LostHati!

I like your idea, and I can see how that would improve the usability of the button. If you're able to figure out, don't hesitate to submit a PR, otherwise I'll look into it myself as soon as I can!

**Comment by dnwjn** [comment]
This is implemented in #52.

Example:

<img width="294" alt="image" src="https://github.com/user-attachments/assets/085b3d54-35e7-4138-ae16-ff1e6180748a" />
